lib/service/webhook.go

```diff
@@ -16,8 +17,14 @@ func (svc *LndhubService) StartWebhookSubscribtion(ctx context.Context, url stri
 	svc.Logger.Infof("Starting webhook subscription with webhook url %s", svc.Config.WebhookUrl)
 	incomingInvoices := make(chan models.Invoice)
 	outgoingInvoices := make(chan models.Invoice)
-	svc.InvoicePubSub.Subscribe(common.InvoiceTypeIncoming, incomingInvoices)
-	svc.InvoicePubSub.Subscribe(common.InvoiceTypeOutgoing, outgoingInvoices)
+	_, err := svc.InvoicePubSub.Subscribe(common.InvoiceTypeIncoming, incomingInvoices)
```

Comment:
This is just a bit cleaning up, nothing to do with the PR. Errors should always be checked.